modules/rubiconBidAdapter.js

```diff
@@ -1116,11 +967,15 @@ function mapSizes(sizes) {
 export function classifiedAsVideo(bidRequest) {
   let isVideo = typeof deepAccess(bidRequest, `mediaTypes.${VIDEO}`) !== 'undefined';
   let isBanner = typeof deepAccess(bidRequest, `mediaTypes.${BANNER}`) !== 'undefined';
+  let isBidOnMultiformat = typeof deepAccess(bidRequest, `params.bidonmultiformat`) !== 'undefined';
   let isMissingVideoParams = typeof deepAccess(bidRequest, 'params.video') !== 'object';
```

Comment:
![image](https://user-images.githubusercontent.com/13972794/219442385-c46c60a3-67fa-4c97-a2f4-808e3a42b615.png)

Prob fine I guess but if it was null we would not want to consider it having video params. (Not your code just noticed) so no need to change